db/bigtable_eth1.go

```diff
@@ -2033,7 +2033,7 @@ func (bigtable *Bigtable) GetAddressTransactionsTableData(address []byte, pageTo
 			utils.FormatAddressWithLimitsInAddressPageTable(address, t.From, fromName, false, digitLimitInAddressPagesTable, nameLimitInAddressPagesTable, true),
 			utils.FormatInOutSelf(address, t.From, t.To),
 			utils.FormatAddressWithLimitsInAddressPageTable(address, t.To, toName, false, digitLimitInAddressPagesTable, nameLimitInAddressPagesTable, true),
-			utils.FormatAmount(new(big.Int).SetBytes(t.Value), utils.Config.Frontend.ElCurrency, 6),
```

Comment:
**issue:** The method `FormatAmount` is not capable to handle currencies other than what is mentioned here
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/2bf7611b5ef3dba2ea0e0fe116584c3fad65365b/utils/eth1.go#L328

The withdrawals use `FormatClCurrency`, please check whether that can just be used or what changes are required.

Reply:
done